space-cli/cmd/utils/input/input.go

```diff
@@ -0,0 +1,25 @@
+package input
+
+import "github.com/AlecAivazis/survey/v2"
+
+type inputInterface interface {
+	AskOne(p survey.Prompt, respone interface{}, opts ...survey.AskOpt) error
+}
+
+// Input struct for parameter
+type input struct{}
+
+// AskOne calls survey.AskOne
+func (i *input) AskOne(p survey.Prompt, respone interface{}, opts ...survey.AskOpt) error {
```

Comment:
I believe the 3rd parameter is useless. As we are not using it anywhere in the test case. Verify it & if we are not using remote it. Directly pass it to survey.AskOne